Assets/Thirdweb/Examples/Scripts/Prefab_LoadNFT.cs

```diff
@@ -0,0 +1,59 @@
+using System.Collections;
+using UnityEngine;
+using UnityEngine.UI;
+using TMPro;
+using Thirdweb;
+using UnityEngine.Networking;
+
+public class Prefab_LoadNFT : MonoBehaviour
+{
+    [Header("SETTINGS")]
+    public string contractAddress = "0x2e01763fA0e15e07294D74B63cE4b526B321E389";
+    public int tokenID = 0;
+
+    [Header("UI ELEMENTS (DO NOT EDIT)")]
+    public Image nftImage;
+    public TMP_Text nftMetadata;
+
+    private void Start()
+    {
+#if !UNITY_EDITOR
+        OnLoadNFT();
+#endif
+    }
+
+    public async void OnLoadNFT()
+    {
+        try
+        {
+            Contract contract = ThirdwebManager.Instance.SDK.GetContract(contractAddress);
+            NFT nft = await contract.ERC721.Get(tokenID.ToString());
```

Comment:
so we should be able to render ERC1155 nfts through this as well... wonder if that makes `NFT` a better input to this, since it already handles both.

another option would be to either detect the type of contract we're dealing with (would need to be exposed on the unity side) or make the user pass the nft 'type'